src/main/resources/static/js/main.js

```diff
@@ -17,36 +17,43 @@ jQuery(document).ready(function() {
 	var frontendServiceUrl = $('#frontendServiceUrl').text();
 
 	function loadMainPage() {
+		$("#navbarResponsive").removeClass("show");
```

Comment:
Why is the indention here ...
```
Line one
        Line 2
    Line 3
        Line4
```
?

Reply:
It's caused by the use of tab in the source code... it hasn't been formatted correctly.